### ISSUE TITLE
Error when trying to add new field to pod

### ISSUE THREAD

**Issue by axettone** [pods-framework/pods#3859]
Hi! I successfully create a pod and added two fields. After this first stage, if I add another field, when I press the "Save Pod" button, I get this error: 
"<fieldname> is reserved for internal WordPress or Pods usage, please try a different name".

**Comment by jimtrue** [comment]
What's the field name you're adding? The message above is fairly clear that you're using a reserved WordPress name for your field that you can't do.

**Comment by axettone** [comment]
The name is "provincia", which I don't think is a reserved name 😀

Inviato da iPhone

**Comment by axettone** [comment]
Ok, I found the issue. When I add a new field, I have to specify a name that is not used as name for an entity. My suggestion is to add this to the help tip.
![schermata 2016-11-14 alle 13 29 36](https://cloud.githubusercontent.com/assets/11018582/20264945/67b35ad4-aa6e-11e6-90ca-3dd310efdd90.png)

**Comment by JoryHogeveen** [comment]
@axettone 
Nice tip. A lot of tooltips don't have content yet so I think 2.7 should have this all done.
I'm waiting for some PR's to get merges, once this step is made I'll continue on this.

**Comment by anitagraham** [comment]
trying out pods.  Got this same error when creating a new with several fields.  Deleted the field called 'name' and the pod saved correctly.

Surely _everything_ can have a 'name' field.

**Comment by jimtrue** [comment]
Believe it or not, name is a reserved field name, @anitagraham https://codex.wordpress.org/Reserved_Terms

**Comment by jimtrue** [comment]
Related #4122

**Comment by sc0ttkclark** [comment]
Resolved in 2.8
